File: fibonacci.py
def fibonacci(n): 
    fib = [];
    first = 0;
    sec = 1;
    fib.append(first);
    fib.append(sec);
    if (n <= 0):
        fib = [];
    elif(n <= 1):
        fib = [0];
    else:
        while (len(fib) <= n):
            fib.append(fib[first] + fib[sec]);
            first += 1
            sec += 1
    return fib;
print(fibonacci(5))

File: fibonacci.ts
function fibonacci(n: number) {
  let arr = [];
  let first = 0;
  let sec = 1;
  arr.push(first);
  arr.push(sec);
  if (n <= 0) {
    arr = [0];
  } else {
    while (arr.length <= n) {
      arr.push(arr[first] + arr[sec]);
      first++;
      sec++;
    }
  }
  return arr;
}
console.log(fibonacci(10));
